src/components/DeleteComment.jsx

```diff
@@ -0,0 +1,27 @@
+import { removeComment } from "../api";
+
+const user = "weegembump";
+
+function DeleteComments({ comment_id, setComments, author }) {
+  if (user === author) {
+    return (
+      <button
+        id="delete_btn"
+        onClick={() => {
+          removeComment(comment_id);
+          document.getElementById("delete_btn").disabled = true;
+          setComments((currentComments) => {
```

Comment:
do you want to optimistically render the deletion of the comment here? if so that's fine, but its one of those cases where maybe the user wants to know for sure that the comment has been removed from the db